### DISCUSSION TITLE
how evaluate MROAs and ROAs

### DISCUSSION THREAD

**Discussion by yangliuming** [sibylhe/mmm_stan#34] [Q&A]
1.Whether MROAS thresholds for each channel can be calculated? if ok. How to calculate the MROAS threshold for each channel?
2. if MROAs are larger than ROAs,whic not make sense?how evaluate MROAs and ROAs
3. Is it possible to suggest the optimal investment portfolio only through the MMM model?
As a student, I am full of doubts about MMM, thank you

**Comment by sibylhe** [comment]
1. If you mean the saturation point of spending, yes. Plug the coefficients into the Hill function. 
2. mROAS can be greater than/equal to/less than ROAS. Definition of ROAS and mROAS is discussed in the article.
ROAS measures past performance, $1 spend has brought how many $ sales.
Marginal ROAS indicates future potential, the return of incremental spending based on current spending. For example, I've spent $100 on SEM, the next $1 spend in SEM will bring how many $ sales.
3. The model result has to be validated. 
If MMM suggests ROAS of SEM is 5, how to prove it's real? You will need to run A/B test on two identical stores. For store A, turn off the SEM channel. Then compare the sales discrepancy of the two stores, the incremental sales of store B is lead by SEM, so the real ROAS of SEM can be calculated: SEM ROAS = (store B sales - store A sales) / store A SEM spending. If the model suggested ROAS aligns with this real ROAS, it is true.
If the model result is validated (by A/B testing, aligns with existing findings/domain knowledge), yes.
